service-front/module/Application/view/application/pages/identity_check_passed.twig

```diff
@@ -10,34 +11,33 @@
-                    </h1>
-                </legend>
-                <div class="govuk-radios" data-module="govuk-radios">
-                    <div class="govuk-radios__item">
-                        <input
-                                class="govuk-radios__input"
-                                id="yes"
-                                name="assistance_on_call"
-                                type="radio"
-                                value="Yes"
-                        >
-                        <label class="govuk-label govuk-radios__label"
-                               for="yes">Yes</label>
-                    </div>
-                    <div class="govuk-radios__item">
-                        <input
-                                class="govuk-radios__input"
-                                id="no"
-                                name="assistance_on_call"
-                                type="radio"
-                                value="No"
-                        >
-                        <label class="govuk-label govuk-radios__label"
-                               for="no">No</label>
-                    </div>
+                <div class="govuk-form-group">
+                    <fieldset class="govuk-fieldset" aria-describedby="contact-hint">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 class="govuk-fieldset__heading">
+                                Did the donor have assistance from a friend or family member on this call?
+                            </h1>
+                        </legend>
+                        <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" id="assistance-1" name="assistance" type="radio" value="no" data-aria-controls="conditional-assistance-1">
+                            <label class="govuk-label govuk-radios__label" for="contact-2">
+                                No
+                            </label>
+                        </div>
+                        <div class="govuk-radios" data-module="govuk-radios">
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="assistance-2" name="assistance" type="radio" value="yes" checked data-aria-controls="conditional-assistance-2">
+                                <label class="govuk-label govuk-radios__label" for="contact">
+                                    Yes, give details
+                                </label>
+                            </div>
+                            <div class="govuk-radios__conditional" id="conditional-assistance">
+                                <div class="govuk-form-group">
+                                    <textarea class="govuk-textarea" rows="2" id="details" name="details" spellcheck="false" aria-describedby="details"></textarea>
```

Comment:
These needs a label. If there's nothing in the design then something like aria-label="Details of the assistance provided" should do.

I think aria-describedby can be removed, since it's currently referring to itself, and I don't think there's any need to have the spellcheck attribute turned off either.